js/app.js

```diff
@@ -6,15 +6,29 @@ const Cart = function(items) {
   this.items = items;
 };
 
+
 Cart.prototype.addItem = function(product, quantity) {
   // TODO: Fill in this instance method to create a new CartItem and add it to this.items
+    let cartItem = new CartItem ();
```

Comment:
This looks good, but you still want to add the product and the quantity for the cartItem in this method.